example/lambdas.yml

```diff
@@ -156,7 +156,6 @@ SnsS3Test:
   source: lambdas/snsS3Test/
   launchInVpc: true
   envs:
-    stackName: '{{stackName}}'
```

Comment:
Is this unnecessary because the stackName is automatically injected as an env var to our Lambdas by our Clouformation template?

Reply:
Yes and that's what breaks the redeploy tests